app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  around_action :unscope_resources
+  load_resource :user
+
+  def show
+    @course_users = @user.course_users
```

Comment:
Shall we add the scope `#with_course_statistics` since you need those data for display?